ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -207,61 +193,50 @@ public void onInitializeSuccess() {
           public void onInitializeError(String errorMessage) {
             Log.d(TAG, "SDK init failed: " + VungleBannerAdapter.this);
             VungleManager.getInstance().removeActiveBannerAd(mPlacementId);
-            VungleListener listener = mVungleListener.get();
-            if (mPendingRequestBanner && listener != null) {
-              listener.onAdFailedToLoad(AdRequest.ERROR_CODE_INTERNAL_ERROR);
+            if (mPendingRequestBanner) {
```

Comment:
should we instead create a new function for checking and use elsewhere, add both checks if(mPendingRequestBanner || VungleManager.getInstance().isBannerAdActive)

Reply:
both fine, this PR ensure to not make big difference with AdMob PR